process_nwb/wavelet_transform.py

```diff
@@ -152,8 +164,71 @@ def wavelet_transform(X, rate, filters='rat', hg_only=True, X_fft_h=None, npad=N
     return Xh, X_fft_h, cfs, sds
 
 
+class ChBandIterator(AbstractDataChunkIterator):
```

Comment:
What's "Ch" stand for? Chunk{ed}? I'd just expand to the whole word. Ch could also be channels.

Reply:
Actually, maybe it is Channel (I'd still expand the word).

Reply:
Yes, it's Channel, and I'll write the full word out